srcpkgs/gallery-dl/template

```diff
@@ -1,22 +1,19 @@
 # Template file for 'gallery-dl'
 pkgname=gallery-dl
-version=1.23.2
-revision=2
+version=1.24.2
+revision=1
 build_style=python3-module
+make_check_args="--ignore test/test_results.py"
 hostmakedepends="python3-setuptools"
 depends="python3-requests"
-checkdepends="python3-requests"
+checkdepends="${depends} youtube-dl python3-pytest"
```

Comment:
Is it possible to use yt-dlp instead?

Reply:
Looking at the tests, they have separate tests for both `youtube-dl` and `yt-dlp`. But running `yt-dlp` tests fail with pytest and need to use `make test` as the template has currently.

Reply:
That's unfortunate, thank you for looking into this